src/moin/scripts/migration/moin19/_utils19.py

```diff
@@ -8,8 +8,8 @@
 import re
 
 from moin.constants.keys import NAME, ACL, CONTENTTYPE, MTIME, LANGUAGE
+from moin.constants.contenttypes import CHARSET
```

Comment:
well, maybe nitpicking, but guess that should be separated, like CHARSET19.

imagine we ever change the CHARSET, the CHARSET19 should stay same, right?

Reply:
fixed in next patch